RELEASE.md

```diff
@@ -7,6 +7,7 @@
 * Added ability and option to initialize site_info with preloaded and formatted wind and solar resource data
 * Bug fix in load following heuristic method: only using beginning of variable load signals 
 * Feature add: added alternative method to defining site boundary.
+* Updated PySAM version from 4.2.0 to 6.0.1. Main changes noted in [PR #425](https://github.com/NREL/HOPP/pull/425)
```

Comment:
I'd add another sentence or couple of bullets to outline the major points relevant to users, then relegate details to the PR.